moveit_ros/planning/trajectory_execution_manager/cfg/TrajectoryExecutionDynamicReconfigure.cfg

```diff
@@ -7,5 +7,6 @@ gen.add("execution_duration_monitoring", bool_t, 1, "Monitor the execution durat
 gen.add("allowed_execution_duration_scaling", double_t, 2, "Accept durations that take a little more time than specified", 1.1, 1, 10)
 gen.add("allowed_goal_duration_margin", double_t, 3, "Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling)", 0.5, 0.1, 5)
 gen.add("execution_velocity_scaling", double_t, 4, "Multiplicative factor for execution speed", 1, 0.1, 10)
+gen.add("allowed_start_tolerance", double_t, 5, "Allowed joint-value tolerance for validation of trajectory's start point against current robot state", 1e-3, 0);
```

Comment:
On Tue, Aug 23, 2016 at 10:17:28AM -0700, Dave Coleman wrote:
> if this is a tricky parameter to tune we should document it for others to tune themselves per robot.
> i'd recommend a new tutorial named "Tuning Magic Numbers for Real Robots" and list all the parameters in our various yaml and roslaunch files

I would actually prefer a non-breaking default that can be changed to be *more restrictive*, if requested.
Thus nobody would see his setup broken with the default configuration because of this safety feature.